USER_GUIDE.md

```diff
@@ -0,0 +1,160 @@
+- [User Guide](#user-guide)
+    - [Getting Started](#getting-started)
+    - [Connecting](#connecting)
+    - [Indexing](#indexing)
+    - [Getting a document](#getting-a-document)
+    - [Searching for documents](#searching-for-documents)
+    - [Falling back to OpenSearch.Net](#falling-back-to-opensearchnet)
+  - [OpenSearch.Net](#opensearchnet)
+    - [Getting Started](#getting-started-1)
+    - [Connecting](#connecting-1)
+# User Guide
+
+This user guide specifies how to include and use the .NET client in your application.
+
+### Getting Started
+
+Include OSC in your .csproj file.
+```xml
+<Project>
+  ...
+  <ItemGroup>
+    <PackageReference Include="Osc" Version="1.0.0" />
```

Comment:
Maybe `<PackageReference Include="opensearch.client" Version="1.0.0" />`?

Reply:
Thank you. Addressed in 8ddc23a2be05d799ebaa96f6ec557553174aab4e